src/Management/src/Endpoint/Actuators/Health/HealthEndpointOptions.cs

```diff
@@ -8,9 +8,29 @@ namespace Steeltoe.Management.Endpoint.Actuators.Health;
 
 public sealed class HealthEndpointOptions : EndpointOptions
 {
-    public ShowDetails ShowDetails { get; set; }
+    /// <summary>
+    /// Gets or sets whether to show components in responses.
+    /// </summary>
+    public ShowValues ShowComponents { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether to show details of components in responses.
+    /// </summary>
+    public ShowValues ShowDetails { get; set; }
+
+    /// <summary>
+    /// Gets or sets the claim requirements for using this endpoint.
```

Comment:
```suggestion
    /// Gets or sets the claim requirements for retrieving components and/or details.
```
From what I understand, this setting doesn't affect whether the endpoint can be used.

Reply:
We may as well just take what's in the docs:

![image](https://github.com/user-attachments/assets/837acf22-36fe-4d9a-b8eb-d78d65072fdc)